cypress/support/elements/portal-report/activity-score-settings.js

```diff
@@ -0,0 +1,79 @@
+import { getByCypressTag } from "../../../utils";
+
+class ActivityScore {
+
+  verifyActivityScoreSettingsOptionLabel(setting, label) {
+    const option = ["No score", "Manual", "Rubric", "MCQ Scored"];
```

Comment:
I'm wondering whether this ought to be defined in the class rather than in each method; though it's only used 3 times, so it's debatable.